Shadowlands/Covenants.lua

```diff
@@ -2400,14 +2411,22 @@ elseif baseClass == "WARRIOR" then
 
             indicator = function () if cycle_for_condemn then return "cycle" end end,
 
+            timeToReady = function()
+                -- Instead of using regular resource requirements, we'll use timeToReady to support the spend system.
+                if state.spec.fury or rage.current >= 20 then return 0 end
+                return rage.time_to_20
+            end,
+
             handler = function ()
                 applyDebuff( "target", "condemned" )
 
                 if not state.spec.fury and buff.sudden_death.down then
-                    local extra = min( 20, rage.current )
+                    local extra = min( 40, rage.current ) -- Can spend up to 40 rage max
```

Comment:
This isn't meant to reflect the entire rage cost; only the excess rage that wasn't spent in `spendResources` using `spend` and `spendType`.

Reply:
Yes, but `spend` is always 0 (unless fury rage generator) to purposely get evaluated in `timeToReady`. It follows the same flow as Execute does since it's an ability replacement.

Am I wrong in my thinking here?

Reply:
But putting the actual cost in TimeToReady actually accomplishes that already.  I think you might be reacting to a resource caching but that I fixed.  You're putting a bandaid on something that probably isn't a problem in the next release.

Reply:
I reverted a majority of the Condem ability changes from the original PR and then made a commit for just updating the handler for the new DF talent interactions. Thanks for considering.